src/Eccube/Application.php

```diff
@@ -282,40 +282,44 @@ public function initRendering()
         $this->register(new \Silex\Provider\TwigServiceProvider(), array(
             'twig.form.templates' => array('Form/form_layout.twig'),
         ));
-        $this['twig'] = $this->share($this->extend('twig', function (\Twig_Environment $twig, \Silex\Application $app) {
+        $this['twig'] = $this->share($this->extend('twig', function(\Twig_Environment $twig, \Silex\Application $app) {
             $twig->addExtension(new \Eccube\Twig\Extension\EccubeExtension($app));
             $twig->addExtension(new \Twig_Extension_StringLoader());
 
             return $twig;
         }));
 
-        $this->before(function (Request $request, \Silex\Application $app) {
+        $this->before(function(Request $request, \Silex\Application $app) {
             // フロント or 管理画面ごとにtwigの探索パスを切り替える.
-            $app['twig'] = $app->share($app->extend('twig', function (\Twig_Environment $twig, \Silex\Application $app) {
+            $app['twig'] = $app->share($app->extend('twig', function(\Twig_Environment $twig, \Silex\Application $app) {
                 $paths = array();
 
                 // 互換性がないのでprofiler とproduction 時のcacheを分離する
 
                 if (isset($app['profiler'])) {
-                    $cacheBaseDir = __DIR__ . '/../../app/cache/twig/profiler/';
-                }else{
-                    $cacheBaseDir = __DIR__ . '/../../app/cache/twig/production/';
+                    $cacheBaseDir = __DIR__.'/../../app/cache/twig/profiler/';
+                } else {
```

Comment:
__[phpmd]__
* The method initRendering uses an else expression. Else is never necessary and you can simplify the code to work without else.